NuRadioReco/modules/io/snowshovel/AriUtils_dicts.py

```diff
@@ -0,0 +1,90 @@
+from NuRadioReco.modules.base.module import setup_logger
+logger = setup_logger("AriUtils_dicts")
+
+# script containing dictionaries defined in snowshovel software
+# copied from snowshovel/scripts/online/AriUtils.py
```

Comment:
Can you add a link to the snowshovel software (even if it is hidden behind a password)

Reply:
I actually think it is ok to have it all public, but full disclosure that we didn't come up with this alone is probably good.